relayer/src/services/polygon-zkevm.ts

```diff
@@ -0,0 +1,322 @@
+import { readFileSync } from "fs";
+
+import { PolygonZKEvmBridge__factory } from "../../types-gen/contracts";
+import { parseBridgeEvent } from "../common/events/polygon-zkevm/BridgeEvent";
+import { sleep } from "../common/util";
+import type { TransactionHash } from "./types";
+import { BaseRelayerService } from "./types";
+
+const CONTRACTS: Record<number, NetworkContracts> = {
+  1101: {
+    l1: {
+      PolygonZKEvmBridge: "0x2a3DD3EB832aF982ec71669E178424b10Dca2EDe",
+    },
+    l2: {
+      PolygonZKEvmBridge: "0x2a3DD3EB832aF982ec71669E178424b10Dca2EDe",
+    },
+    bridgeServiceUrl: "https://bridge-api.zkevm-rpc.com",
+  },
+  1442: {
+    l1: {
+      PolygonZKEvmBridge: "0xF6BEEeBB578e214CA9E23B0e9683454Ff88Ed2A7",
+    },
+    l2: {
+      PolygonZKEvmBridge: "0xF6BEEeBB578e214CA9E23B0e9683454Ff88Ed2A7",
+    },
+    bridgeServiceUrl: "https://bridge-api.public.zkevm-test.net",
+  },
+};
```

Comment:
weren't we going to introduce a conf file for this?

Reply:
yes, so we are hardcoding them in a JS file instead of having them in a constants file somewhere?

Reply:
They will anyways be used only in this file, so i didn't see the point of moving them to a constants file.